tasks/deploy/constants/optimismConfig.ts

```diff
@@ -54,13 +54,13 @@ const optimismConfig = {
     "0x8aE125E8653821E851F12A49F7765db9a9ce7384", // DOLA
```

Comment:
@0xMotto actually let's remove this whole file and any file that uses it, sine it's not used anymore and it will only confuse people